packages/x-charts/src/LineChart/LineChart.tsx

```diff
@@ -43,6 +50,10 @@ export interface LineChartProps
   tooltip?: ChartsTooltipProps;
   axisHighlight?: ChartsAxisHighlightProps;
   legend?: ChartsLegendProps;
+  /**
+   * Render the line highlight item if `true`.
```

Comment:
What do you think about following the convention for boolean prop JSDoc employed by the pickers? 🤔 
```suggestion
   * If `true`, render the line highlight item.
```